src/components/posts/comments/comment-element.svelte

```diff
@@ -1,36 +1,79 @@
 <!--Comment-Element for the comment-Modal-Window. Shows one comment with the user and the text-->
 <script lang="ts">
-	import { Jonas } from '$images';
+	import type { Comment } from '$domains';
+	import { page } from '$app/stores';
+	import { goto } from '$app/navigation';
 	export let id: string;
-	export let comment: string;
-	import { Avatar } from '@skeletonlabs/skeleton';
-	let shortComment: string = comment.substring(0, 128);
+	export let comment: Comment;
+	import { Avatar, getModalStore } from '@skeletonlabs/skeleton';
+	import { Person } from '$images';
+	import { calculatePassedTime } from '$utils';
+
+	let shortComment: string = comment.content.substring(0, 64);
 	let moreClicked = false;
+	let dateString: string = calculatePassedTime(comment.creationDate);
+	const modalStore = getModalStore();
+
 	//function to toggle wether the comment is shown fully or not (only for long comments)
 	function changeMoreClicked() {
 		moreClicked = !moreClicked;
 	}
+
+	function getWords(text: string) {
+		const words = text.split(' ');
+		return words;
+	}
 </script>
 
 <!--Grid with two columns. The left Column contains the profile picture and the right the comment-->
 <div class="grid grid-cols-[auto_1fr] gap-2 m-2">
 	<!--Left column-->
-	<Avatar src={Jonas} width="w-8 md:w-10 min-w-0" />
+	<Avatar
+		src={comment.author.profilePictureUrl ? comment.author.profilePictureUrl : Person}
+		width="w-8 md:w-10 min-w-0"
+	/>
 	<!--right column-->
 	<div
 		class="card p-2 variant-soft-tertiary dark:variant-soft-surface rounded-tl-none space-y-2 min-w-0"
 	>
 		<!--Header containing the username and nickname and the time that has passed since the post-->
 		<header class="flex justify-between items-center pr-1">
-			<p class="text-xl md:text-2xl font-bold">Username</p>
-			<small class="text-sm md:text-base">12.12.2023 17:45 Uhr</small>
+			<button
+				class={`${'hover:text-gray-400'}`}
+				on:click={() => {
+					const currenPath = $page.url.pathname.split('/')[1];
```

Comment:
Du hast wohl main falsch in deine Branch gemerged... Das ist wahrscheinlich nicht der einzige Fehler :(